.changeset/867-prettier-as-direct-dependency.md

```diff
@@ -0,0 +1,20 @@
+---
+issue: https://github.com/pmedianetwork/adverity-presense-frontend/issues/867
+type: minor
```

Comment:
I think we settled on a `major` release, given that we have a Breaking changes section here :)

```suggestion
type: major
```